source/_static/calendar.csv

```diff
@@ -1,43 +1,43 @@
 Date,Lecture / Activity,Assignments / Reading
 ,**Week 1**,
-M 1/5,:slides:ref:`00_intro`,
-W 1/7,:slides:ref:`01_linux_basics`,:ref:`Ch. 1 <readings>`
-F 1/9,":slides:ref:`02_linux_basics`",:ref:`Ch. 2 and pg 926-930 <readings>`
+M 1/4,Welcome to CS312,
+W 1/6,Distribution History,
+F 1/8,Linux Basics (Day 2),
```

Comment:
`Distribution History` to `Linux Basics (Day 2)` seems more discontinuous than it is.